src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const clickPromis = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject('Promise was rejected!');
+  }, 3000);
+});
+
+const div = document.createElement('div');
+
+document.body.append(div);
```

Comment:
It's good practice to add elements to the DOM after you've finished modifying them. In this case, you're appending the `div` to the body before adding classes and setting the text content. This can cause unnecessary reflows and repaints. Consider moving this line to after you've finished modifying the `div`.